theme/plugins/objects/server/enemy1-object.js

```diff
@@ -5,26 +5,22 @@
  */
 
 const { EnemyObject } = require('reldens/lib/objects/server/object/type/enemy-object');
-const { Logger, sc } = require('@reldens/utils');
+const { sc } = require('@reldens/utils');
 
 class Enemy1Object extends EnemyObject
```

Comment:
I though we were going to remove both classes at the end, right? enemy1-object and enemy2?